databases/functions/userFollowUser/get-all-following-user.js

```diff
@@ -0,0 +1,18 @@
+/**
+ *
+ * @param {Model} userFollowUserModel
+ * @param {RegisterBodyData.Users} users
+ */
+module.exports = async (userFollowUserModel, userIdFollower, _transaction = null) => {
+  const followingUsers = await userFollowUserModel
+    .findAll({
+      where: {
+        user_id_follower: userIdFollower
+      },
+      attributes: ['user_id_followed'],
+      raw: true
+    })
+    .then((users) => users.map((user) => user.user_id_followed));
+
+  return followingUsers;
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new function `module.exports` in `get-all-following-user.js` seems to be well implemented. It correctly uses the Sequelize model to find all users that the current user is following and returns an array of their IDs. However, it's important to handle potential errors that might occur during the database query. Adding a try-catch block would improve error handling.

```diff
module.exports = async (userFollowUserModel, userIdFollower, _transaction = null) => {
+   try {
    const followingUsers = await userFollowUserModel
      .findAll({
        where: {
          user_id_follower: userIdFollower
        },
        attributes: ['user_id_followed'],
        raw: true
      })
      .then((users) => users.map((user) => user.user_id_followed));

    return followingUsers;
+   } catch (error) {
+     console.error(error);
+     throw error;
+   }
};
```
This change will ensure that any errors that occur during the execution of the database query are caught and properly handled.


<!-- This is an auto-generated comment by OSS CodeRabbit -->